tasks/README.md

```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task721_hate_speech18_classification` | Classify sentiment as nohate, hate, skip or neutral from statement | Classification
+`task722_DSTC2_classification` | Classify if the speaker is a customer or restaurant system  | Classification  
+`task723_DSTC2_classification` | Determine the customer goals from the given customer side of conversation  | Classification  
+`task724_DSTC2_classification` | Classify the experience of the speaker understanding in terms of strongly agree, agree,slightly agree,slightly disagree and strongly disagree. | Classification  
```

Comment:
Maybe change to: `Given the responses by a restaurant system, classify the experience of the customer understanding.`
Please consider rewriting the summaries in this format.